src/inference/src/dev/threading/istreams_executor.cpp

```diff
@@ -553,5 +553,109 @@ IStreamsExecutor::Config IStreamsExecutor::Config::reserve_cpu_threads(const ISt
     return config;
 }
 
+IStreamsExecutor::Config IStreamsExecutor::Config::update_executor_config(
```

Comment:
Please add smoke test cases for this feature

Reply:
updated